index.bs

```diff
@@ -201,9 +201,9 @@ The <dfn method for="StorageBucketManager">delete(|name|)</dfn> method steps are
 
```

Comment:
(I don't know why I can't comment on L200)

Does the reject on L200 also need to be updated? Perhaps also moved out of the "in parallel" part?

Same question for the `persist()` steps, both 1 and 2 under "in parallel". But I'm not sure. Maybe it should be "queue a storage task to run these steps" instead of "run these steps in parallel".

Reply:
Thanks for the catch. Unfortunately looks like the example I was looking at for "in parallel" use is also wrong here which hasn't been helpful :(. Adopted "queue a storage task to run these steps" since looking at other examples, looks like is right to use here.

Reply:
great. I think the first step (validating bucket name) belongs outside the storage task steps as well. At least, that's what's implemented in Chromium and I don't see a need to go to the storage task queue because name validation is a static operation. WDYT?

Reply:
Oops, missed that part of the comment. Moved name validation above task steps.